scripts/calculator.js

```diff
@@ -24,4 +24,38 @@ const state = {}
 */
 const updateScreen = (screenText = '') => {
   document.getElementById('screen').innerText = screenText
-}
\ No newline at end of file
+}
+
+/*
+  This function will set up all of the "event listeners" we'll need in order
+  for the calculator to works. To reiterate: an "event" is some action a user
+  performs (clicks, scrolling, mouse movement, keyboard use etc). 
+  
+  The "listeners" we're going to set up here will listen for a type of event,
+  and when they detect it they'll call some sort of function we specify.
+
+  To clarify, this will work in a series of steps:
+  1. "Declare" the initializeListeners function. (this is literally what 
+  we're doing in the function below).
+  2. "Declare" a second, "listening" function. We'll call it clickListener.
+  clickListener is what will get run when a click occurs.
+  3. "Call" initializeListeners, which will actually run the function and attach
+  the "listener" to the calculator buttons.
+  4. Click the button, and see clickListener get "called".
+*/
+
+// 2.
+const clickListener = () => {
```

Comment:
Notice how the number "1" is all this "listening" function deals with. 

The trouble is that all of our buttons will need to do different things in order to work. The easy (but slow) way to do it would be to create unique a "listening" function for every single button. There are clever ways to avoid doing that, but I'll leave that up to you.